presto-common/src/main/java/com/facebook/presto/common/RuntimeMetricName.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.common;
+
+/**
+ * Names for RuntimeMetrics used in the core presto code base.
+ * Connectors could use arbitrary metric names not included in this class.
+ */
+public class RuntimeMetricName
+{
+    public static final String DRIVER_COUNT_PER_TASK = "driverCountPerTask";
+    public static final String TASK_ELAPSED_TIME_NANOS = "taskElapsedTimeNanos";
```

Comment:
small nit: I see the values in millisecs `ms` , but the metric name says `nanos` ?

Reply:
The value is stored as integer nanos in the metric. Thus, it is better to carry this information in the name. But we are rendering it with more human-readable time format. In the coordinator UI, it is rendered with formatDuration() function. In the cli tool it is rendered by the formatTime() function. The ms in the example snapshot is because it is less than a second. It could be shown as XmYs for longer time.